### ISSUE TITLE
requested bucket name is not available in converge projects CI

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-example-foundation#716]
### TL;DR

In several runs of the CI, "requested bucket name is not available" during converge projects

### Observed behavior

       Error: googleapi: Error 409: The requested bucket name is not available. The bucket namespace is shared by all users of the system. Please select a different name and try again., conflict
       
         with module.bu1_shared.module.infra_pipelines.google_storage_bucket.pipeline_infra["cloudbuild"],
         on ../../../4-projects/modules/infra_pipelines/main.tf line 49, in resource "google_storage_bucket" "pipeline_infra":
         49: resource "google_storage_bucket" "pipeline_infra" {

### Terraform Configuration

```hcl
CI
```

### Terraform Version

```sh
CI
```
